tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -4646,150 +4677,105 @@ TEST_F(OpConverterTest, ConvertPool3D) {
     std::vector<int> strides;
     string padding;
     string data_format;
-    bool is_max_pooling;
     std::vector<int> expected_output_dims;
-    std::vector<float> expected_output;
+    // output vector: {MaxPool2D, AvgPool2D, MaxPool3D, AvgPool3D }
+    std::vector<std::vector<float>> expected_output;
   };
 
   // Start here
   const std::vector<float> common_array{-4, 2,  15, 3, 6,   -3, 22, 1,   88,
```

Comment:
Do we want to rename "common_array" to "common_input", and "common_array_2d" to "actual_input_for_2d_op"?
Also, it would be very helpful to add a comment, something like this:
// We use common_input as the input to test both 2D and 3D pooling operations, to simplify TestParams. For 2D operations, only the first 1/3 of the values in common_input, that is the values in actual_input_for_2d_op,  are actually used.

Reply:
- Common_array is used for both input and expected output (for 3D ops), nevertheless common_input sounds better, therefore I have changed that.
- The common_array_2d is actually only used as an output. Renamed to common_2d_output.
- Added comments.